jmx/tests/org.jboss.tools.jmx.jolokia.test/src/org/jboss/tools/jmx/jolokia/test/util/JolokiaTestEnvironmentSetup.java

```diff
@@ -74,10 +75,12 @@ public static void start() throws Exception{
 			int port = EnvTestUtil.getFreePort();
 			jettyServer = new Server(port);
 			SecurityHandler securityHandler = createSecurityHandler();
-			ServletContextHandler jettyContext = new ServletContextHandler(jettyServer, "/", null, securityHandler, null, null);
+			ServletContextHandler jettyContext = new ServletContextHandler(jettyServer, "/"); //, null, securityHandler, null, null);
```

Comment:
had to remove the security handler because if not, tests are failing with 
java.io.IOException: org.jolokia.client.exception.J4pRemoteException: Forbidden
	at org.jboss.tools.jmx.jolokia.test.internal.connection.JolokiaMBeanServerConnectionGetDomainsTest.testGetDomains(JolokiaMBeanServerConnectionGetDomainsTest.java:25)
Caused by: org.jolokia.client.exception.J4pRemoteException: Forbidden
	at org.jboss.tools.jmx.jolokia.test.internal.connection.JolokiaMBeanServerConnectionGetDomainsTest.testGetDomains(JolokiaMBeanServerConnectionGetDomainsTest.java:25)